docker/Dockerfile.wasm32-unknown-emscripten

```diff
@@ -17,5 +17,7 @@ RUN apt-get update && apt-get install --assume-yes --no-install-recommends \
 
 ENV CROSS_TOOLCHAIN_PREFIX=em
 ENV CROSS_SYSROOT=/emsdk/upstream/emscripten/cache/sysroot
-ENV CARGO_TARGET_WASM32_UNKNOWN_EMSCRIPTEN_RUNNER=node \
-    BINDGEN_EXTRA_CLANG_ARGS_wasm32_unknown_emscripten="--sysroot=$CROSS_SYSROOT"
+ENV CROSS_TARGET_RUNNER="node"
```

Comment:
We could possibly provide the additional pthread flags to the runner. See:
https://github.com/emscripten-core/emscripten/blob/e9ba3598df8f219b8c9c505c4ed1ed4618001123/emcmake.py#L44